nautobot/extras/datasources/git.py

```diff
@@ -403,6 +405,20 @@ def import_config_context(context_data, repository_record, job_result, logger):
         data = context_data.copy()
         del data["_metadata"]
 
+        if context_metadata["schema"]:
+            if getattr(context_record, "name", None) != context_metadata["schema"]:
```

Comment:
Is this correct? Seems suspect to me. Shouldn't it be `context_record.schema` rather than `context_record.name`?